mm2src/mm2_bitcoin/chain/src/output.txt

```diff
@@ -0,0 +1,7 @@
+
+running 1 test
+**is_zcash: true
+test block_header::tests::test_rick_v1_block_header_des ... ok
+
+test result: ok. 1 passed; 0 failed; 0 ignored; 0 measured; 45 filtered out; finished in 0.00s
+
```

Comment:
What is this for?

Reply:
It's a log file that I accidentally pushed :sweat_smile: